metadata-db-app/src/cmr/metadata_db/services/messages.clj

```diff
@@ -175,3 +178,43 @@
   (format (str "Variable association could not be associated with provider [%s]. "
                "Variable associations are system level entities.")
           provider-id))
+
+(defn concept-not-found [provider-id field-name field-value]
+  (format "Unable to find saved concept for provider [%s] and %s [%s]"
+          provider-id
+          field-name
+          field-value))
+
+(defn concept-higher-transaction-id
+  [revision-id concept-id transaction-id this-revision-id this-transaction-id]
+  (format (str "Revision [%d] of concept [%s] has transaction-id [%d] "
+               "which is higher than revision [%d] with transaction-id [%d].")
+          revision-id
+          concept-id
+          transaction-id
+          this-revision-id
+          this-transaction-id))
+
+(defn concept-lower-transaction-id
+  [revision-id concept-id transaction-id this-revision-id this-transaction-id]
+  (format (str "Revision [%d] of concept [%s] has transaction-id [%d] "
+               "which is lower than revision [%d] with transaction-id [%d].")
+          revision-id
+          concept-id
+          transaction-id
+          this-revision-id
+          this-transaction-id))
+
+(defn pvn-equality-failure
+  [new-concept old-concept]
+   (format (str "Revision [%d] of concept [%s] has the same provider [%s] "
```

Comment:
This error message is for ingest providers. They wouldn't care about the new concept id and revision id that CMR tries to create but failed. I think the collection style error message is more user friendly.